grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/bazel/rules/KotlinRules.kt

```diff
@@ -58,14 +58,18 @@ fun StatementsBuilder.kotlinCompiler(
     KOTLINC_RELEASE_SHA eq kotlinCompilerReleaseSha.quote()
     newLine()
 
-    KOTLINC_RELEASE eq obj {
-        "urls".quote() eq array(
-            """"https://github.com/JetBrains/kotlin/releases/download/v{v}/kotlin-compiler-{v}.zip".format(v = $KOTLIN_VERSION)"""
-        )
-        "sha256".quote() eq KOTLINC_RELEASE_SHA
-    }
+    load(
+        "@io_bazel_rules_kotlin//kotlin:repositories.bzl",
+        "kotlin_repositories",
+        "kotlinc_version"
+    )
+
+    KOTLINC_RELEASE eq """kotlinc_version(
```

Comment:
Currently there is no migrateToBazel validation on CI, could you also include `migrateToBazel` output as part of this PR?

Reply:
Updated with new changes via `migrateToBazel`